docker/collector/kafka/install.sh

```diff
@@ -20,10 +20,6 @@ apk add --update --no-cache jq curl
 
 APACHE_MIRROR=$(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 | jq -r '.preferred')
 
-curl -sSL $APACHE_MIRROR/zookeeper/zookeeper-$ZOOKEEPER_VERSION/apache-zookeeper-$ZOOKEEPER_VERSION.tar.gz | tar xz
```

Comment:
AFAIK, Zookeeper binaries are not required as they come as part of Kafka distribution (Unless until kip-500 ;) )